### PR TITLE
Update test runner with args support

### PR DESCRIPTION
**Pull request by JanakaSandaruwan**
## Purpose
https://github.com/wso2-enterprise/choreo/issues/30116

### CODE REVIEW

File: en/docs/testing/test-components-with-test-runner.md [hunk 1]
```diff
@@ -94,3 +94,17 @@ You have successfully created a Test Runner component using a buildpack of you c
...
+
+           1. Click the drop-down icon next to **Run Now** and then click **Run with Arguments**. 
+           2. In the **Runtime Arguments** pane that opens, enter the arguments you want to pass to your application. 
+           3. Click **Execute**. This triggers the task with the specified arguments.
+    
+        !!! info
+            For `Postman Collection`:
+
+            -  You can run only a set of collections in the directory by specifying `-f` or `--files` as the first argument and a comma-separated list of collections as the next argument.
+            - `newman` is internally used for running the postman collections. Hence, you can pass its arguments as well.
```
Comment (a5anka) [inline en/docs/testing/test-components-with-test-runner.md:109]: Shall we add a link to the newman project here?

File: en/docs/testing/test-components-with-test-runner.md [hunk 2]
```diff
@@ -94,3 +94,17 @@ You have successfully created a Test Runner component using a buildpack of you c
...
+         If you want to inject dynamic values into your application as command-line arguments when you run a test runner component, follow the steps given below:
+
+           1. Click the drop-down icon next to **Run Now** and then click **Run with Arguments**. 
+           2. In the **Runtime Arguments** pane that opens, enter the arguments you want to pass to your application. 
+           3. Click **Execute**. This triggers the task with the specified arguments.
+    
+        !!! info
+            For `Postman Collection`:
+
+            -  You can run only a set of collections in the directory by specifying `-f` or `--files` as the first argument and a comma-separated list of collections as the next argument.
```
Comment (a5anka) [inline en/docs/testing/test-components-with-test-runner.md:108]: Have we said anywhere the deault behavior is ton run all collections? If not shall we tell that.

File: en/docs/testing/test-components-with-test-runner.md (hunk 1 above)
Comment (JanakaSandaruwan) [reply en/docs/testing/test-components-with-test-runner.md]: Fixed with https://github.com/wso2/docs-choreo-dev/pull/1550/commits/8930e552bd45daa64e69a046d36e94588dfab141

File: en/docs/testing/test-components-with-test-runner.md (hunk 2 above)
Comment (JanakaSandaruwan) [reply en/docs/testing/test-components-with-test-runner.md]: Fixed with https://github.com/wso2/docs-choreo-dev/pull/1550/commits/8930e552bd45daa64e69a046d36e94588dfab141

**Review by a5anka — APPROVED** [review]

**Review by Nashaath — APPROVED** [review]

**Comment by Nashaath** [comment]
Closing since these changes were pushed via https://github.com/wso2/docs-choreo-dev/pull/1555

**Pull request closed without merging**